udata/rdf.py

```diff
@@ -218,6 +220,25 @@ def url_from_rdf(rdf, prop):
         return value.identifier.toPython()
 
 
+def schema_from_rdf(rdf):
+    '''
+    Try to extract a schema from a conformsTo property.
+    Currently the "issued" property is not harvest.
+    '''
+    resource = rdf.value(DCT.conformsTo)
+    if not resource:
+        return None
+
+    schema = Schema()
+    if isinstance(resource, (URIRef, Literal)):
+        schema.url = resource.toPython()
```

Comment:
In the particular case of harvesting, could we imagine setting schema.name if the url namespace is from schema.data.gouv.fr? Or would that lead again to inconsistencies in the end?

Reply:
Yes, it's fine I think! Done.